proto/stride/stakeibc/lsm.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package stride.stakeibc;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Stride-Labs/stride/v8/x/stakeibc/types";
+
+enum LSMDepositStatus {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  TRANSFER_IN_PROGRESS = 0;
+  TRANSFER_FAILED = 1;
+  DETOKENIZATION_QUEUE = 2;
+  DETOKENIZATION_IN_PROGRESS = 3;
+  DETOKENIZATION_FAILED = 4;
+}
+
+message LSMTokenDeposit {
+  string denom = 1;
+  string chain_id = 2;
+  string validator_address = 3;
+  string amount = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  uint64 record_id = 5;
```

Comment:
If the denom is going to contain the unique ending to the NFT like "cosmosval42" why do we also need this field for the record_id "42".  Won't we always want to look up using the full denom and this would create additional logic to go back and forth from the denom to the record_id alone?

Reply:
Very good question. At one point, there was an ICA we needed to send where we needed to specify the record ID in the transaction. It's possible that it's no longer necessary now that we're not hoarding though - I'll double check the spec!

Reply:
Ah okay - just checked. It was previously used for the `withdrawRewards` transaction (when we were hoarding LSM tokens), but now that we're not, it's not needed.

Good catch! I'll remove